packages/editor-ui/src/components/MainSidebar.vue

```diff
@@ -645,18 +678,181 @@ export default mixins(
 				}
 				return defaultSettingsRoute;
 			},
+			onResize (event: UIEvent) {
+				this.callDebounced("onResizeEnd", { debounceTime: 100 }, event);
+			},
+			onResizeEnd (event: UIEvent) {
+				const browserWidth = (event.target as Window).outerWidth;
+				this.checkWidthAndAdjustSidebar(browserWidth);
+			},
+			checkWidthAndAdjustSidebar (width: number) {
+				if (width < 900) {
+					this.$store.commit('ui/collapseSidebarMenu');
+				}
+			},
 		},
 	});
 </script>
 
+<style lang="scss" module>
+
+$--n8n-logo-text-color: #101330;
```

Comment:
how can we support embed use cases with this?

Reply:
This is result of misunderstanding with Max, logo text should be part of the logo image and not a text element.